### PR TITLE
docs: keyboard maneuver

### PR DESCRIPTION
**Pull request by dstala**
Signed-off-by: Raju Udava <86527202+dstala@users.noreply.github.com>

## Change Summary
Docs update about keyboard shortcuts and general usage

## Change type

- [ ] feat: (new feature for the user, not a new feature for build script)
- [ ] fix: (bug fix for the user, not a fix to a build script)
- [x] docs: (changes to the documentation)
- [ ] style: (formatting, missing semi colons, etc; no production code change)
- [ ] refactor: (refactoring production code, eg. renaming a variable)
- [ ] test: (adding missing tests, refactoring tests; no production code change)
- [ ] chore: (updating grunt tasks etc; no production code change)

## Test/ Verification

Provide summary of changes.

## Additional information / screenshots (optional)

Anything for maintainers to be made aware of

### CODE REVIEW

**Comment by dstala** [comment]
fix needed
- Cmd left navigates to previous route (chrome)

shortcuts to be added
- Create table
- Create column

File: packages/noco-docs/content/en/setup-and-usages/keyboard-maneuver.md [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+
+## Shortcuts
+|       Key | Behaviour               |
+|----------:|:------------------------|
+| `ALT` `t` | Insert new table        |
+| `ALT` `r` | Insert new row          |
+| `ALT` `c` | Insert new column       |
+| `ALT` `f` | Toggle fullscreen mode  |
+| `ALT` `i` | Invite a member to team |
+| `ALT` `,` | Open Settings menu      |
```
Comment (wingkwong) [inline packages/noco-docs/content/en/setup-and-usages/keyboard-maneuver.md:18]: `Open Team & Settings`

File: packages/noco-docs/content/en/setup-and-usages/keyboard-maneuver.md [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+|      Multi Select      |     `↑` `↓` | Move between options               |
+|                        |     `Enter` | Select / deselect option           |
+|        DateTime        |  `Ctrl` `;` | Select current date time           |
+|          Link          |     `↑` `↓` | Move between options               |
+|                        |     `Enter` | Link current selection             |
+|        Checkbox        |     `Enter` | Toggle                             |
+|         Rating         | `<0 ~ Max>` | Enter number to toggle rating      |
+
+
+## Expanded row maneuver
```
Comment (wingkwong) [inline packages/noco-docs/content/en/setup-and-usages/keyboard-maneuver.md:51]: prefer `Expanded Form` over `Expanded Row` as it is using `Expanded Form` in the doc.

**Review by wingkwong — COMMENTED** [review]
Can we keep those title as short as possible?

**Pull request merged**